internal/asyncapi/command_export.go

```diff
@@ -84,7 +84,7 @@ func (c *command) newExportCommand() *cobra.Command {
 	cmd.Flags().Bool("consume-examples", false, "Consume messages from topics for populating examples.")
 	cmd.Flags().String("spec-version", "1.0.0", "Version number of the output file.")
 	cmd.Flags().String("kafka-api-key", "", "Kafka cluster API key.")
-	cmd.Flags().String("schema-context", "default", "Use a specific schema context.")
+	cmd.Flags().String("schema-registry-context", "", "Use a specific Schema Registry context.")
```

Comment:
If we don't have a ticket for supporting flag aliases yet, I'm thinking we should create one. This could also be written as `--sr-context`.